config/configstore/model.go

```diff
@@ -574,15 +611,15 @@ func (acc *Account) GetKeys() (idKeys identity.IDKeys, err error) {
 	}
 	address32Bytes := utils.AddressTo32Bytes(common.HexToAddress(secp256k1.GetAddress(pk)))
 
-	keys[identity.KeyPurposeEthMsgAuth] = identity.IDKey{
+	keys[identity.KeyPurposeEthMsgAuth] = config.IDKey{
```

Comment:
@manuelpolzhofer Should we still use this purpose here? I thought we got rid of it?

Reply:
We still need to rename the purpose constants. It would be a purpose "ACTION_KEY" after the refactoring. If that was the question?